src/script/Monitor/GenerateMONIT.py

```diff
@@ -265,16 +266,40 @@ def execute(self):
 
     # before running make sure no other instance of this script is running
     lockFile = workdir + 'CRAB3_SCHEDD_JSON.Lock'
+    currentPid = str(os.getpid())
```

Comment:
it will be more readable if definition of currentPid is done next to where it is used (line 296)

Reply:
Changed